pkg/cmd/run/root.go

```diff
@@ -124,7 +147,11 @@ var runCmd = &cobra.Command{
 			cobra.CheckErr(err)
 		}
 
-		fmt.Println("Local running, use ctrl-C to stop")
+		spinner.StopMessage("All services are ready to go! 🚀")
+		err = spinner.Stop()
+		cobra.CheckErr(err)
+
+		fmt.Println("\nLocal running, use ctrl-C to stop")
```

Comment:
This might be printed a bit pre-emptively. We could wait by checking the active worker count if we want to make sure workers will actually be available. If things take a bit of time to warm up users may think that this message is an error when they can't reach their functions.

Reply:
Lets make this a new PR